lib/ansible/modules/windows/win_pester.ps1

```diff
@@ -15,6 +15,7 @@ $diff_mode = Get-AnsibleParam -obj $params -name "_ansible_diff" -type "bool" -d
 
 $path = Get-AnsibleParam -obj $params -name "path" -type "str" -failifempty $true
 $tags = Get-AnsibleParam -obj $params -name "tags" -type "list"
+$testParameters = Get-AnsibleParam -obj $params -name "test_parameters" -type "dict"
```

Comment:
Just an FYI, the legacy module wrapper (`Get-AnsibleParam`) does not actually do anything with `-type "dict"`. There's not much you can do about that I just thought it best to mention.

One thing you should change is to follow the `snake_case` convention the module currently has, e.g. change the var to `$test_parameters`. Typically we don't care what format is used as long as it follows what is already in the module.

Reply:
sure, will change them to snake_case.